src/libraries/System.IO.FileSystem/tests/FileStream/FileStreamConformanceTests.cs

```diff
@@ -191,7 +191,12 @@ public async Task WriteByteFlushesTheBufferWhenItBecomesFull()
     public class UnbufferedSyncFileStreamStandaloneConformanceTests : FileStreamStandaloneConformanceTests
     {
         protected override FileOptions Options => FileOptions.None;
+
+#if RELEASE // since buffering can be now disabled by setting the buffer size to 0 or 1, let's test 0 in one config and 1 in the other
+        protected override int BufferSize => 0;
+#else
         protected override int BufferSize => 1;
+#endif
```

Comment:
IMO we don't even need to test BufferSize = 0 more than in the .ctor (verify it doesn't throw) since for `0` and `1` cases, the value is not used in any other place.

Reply:
After I have added this test, I've discovered that `Net5CompatFileStreamStrategy` does not support `bufferSize == 0` and I had to add the `bufferSize == 0 ? 1 : bufferSize` logic when calling its ctor. So I think that there is value in testing both 0 and 1